lib/Utils/PlatformWin.cpp

```diff
@@ -685,6 +689,16 @@ bool Popen(const std::string& Cmd, llvm::SmallVectorImpl<char>& Buf, bool RdE) {
   return !Buf.empty();
 }
 
+std::string Demangle(const std::string& Symbol) {
+  struct AutoFree {
+    const char* Str;
+    AutoFree(const char* Ptr) : Str(Ptr) {}
+    ~AutoFree() { if (Str) ::free((void*)Str); };
```

Comment:
As this deletes the string I'd prefer it to take and keep a `char*`, not a `const char`.